parsers/ccmpredparser.py

```diff
@@ -0,0 +1,28 @@
+from utils.exceptions import InvalidFormat
+from operator import itemgetter
+
+def CCMpredParser(input):
+    contents = input.split('\n')
+
+    output = []
+
+    data = []
+    for line in contents:
+        line = line.lstrip().split('\t')
+        if not line or line[0].isalpha():
+            continue
+        else:
+            data.append(line)
+
+    for res_1, score_array in enumerate(data):
+        for res_2, score in enumerate(score_array):
+            if abs((res_1+1) - int(res_2+1)) >= 5 and score!='':
```

Comment:
You can avoid having to add one to `res_1` and `res_2` if you start to enumerate from 1 instead of 0. You can achieve this passing the start index as the second argument to `enumerate`, for example: `enumerate(data, 1)`